README.md

```diff
@@ -121,7 +121,6 @@ Consider submitting to [the deno.land/x](https://github.com/denoland/registry/bl
 
 ## Chinese
 
-- [deno.js.cn](https://deno.js.cn/)
```

Comment:
No longer working?

Reply:
nop. I just found it's registered by @justjavac . Is there a problem?